src/System.Data.Odbc/src/Common/System/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -0,0 +1,258 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Globalization;
+
+namespace System.Data.Common
+{
+    internal static class DbConnectionStringDefaults
```

Comment:
Thanks! I actually copied them over, so I'm guessing I'll do that. Any special care on the Common folder, or do I just drop the files there?